src/models/validators/CommonSchema.ts

```diff
@@ -1,6 +1,6 @@
 import * as Joi from 'joi';
-import { ValidationErrorItem } from 'joi';
-import { TESTING_ERRORS, VEHICLE_TYPES } from '../../assets/Enums';
+import { TestStationTypes } from "@dvsa/cvs-type-definitions/types/v1/enums/testStationType.enum";
```

Comment:
Appreciate it is a bit of a stop-gap as we do have a ticket in the backlog to move all of the types in this service across to using the types-definition, however I still think it will be worth using the centralised definition for testStationType in `ITestResult` too rather than string, as we are implementing the below in the schema.

```
export interface ITestResult {
  ...
  testStationType: string;
  ...
}
```